app/src/main/shared/styles/buttons.scss

```diff
@@ -1,28 +1,15 @@
+@import "bootstrap";
+
 button, .button {
-    display: inline-block;
-    border: none;
-    background-color: rgb(0, 173, 239);
-    color: white;
-    font-family: inherit;
-    font-size: 12pt;
+    @include button-variant(rgb(0, 173, 239), rgb(200, 200, 200));
+    @include button-size(5px, 20px, 1em, 2em,  0);
     font-weight: bold;
-    padding: 6px 20px 5px 20px;
-    vertical-align: top;
-    outline: rgb(200, 200, 200) outset 1px;
+    @extend .d-inline-block;
+    @extend .align-top;
 }
 
 button[type=submit], .submit{
-    color: #fff;
-    background-color: #5cb85c;
-    outline-color: #4cae4c;
-}
-
-button:hover, .button:hover {
-    outline-width: 2px;
-}
-
-button:active {
-    outline-style: inset;
+    @include button-variant(#5cb85c, #4cae4c);
```

Comment:
one last thing! I do think lets change this rule from just `.submit` to `button.submit, `.button.submit`